src/navigationBar.js

```diff
@@ -150,3 +150,47 @@ export const setupNavbarCollapse = () => {
         navbarCollapseSetup = true;
     }
 }
+
+/**
+ * Helper function to update active navigation elements based on participant type
+ * 
+ * @function updateActiveElements
+ * @param {string} type - The participant filter type
+ */
+export const updateActiveElements = (type) => {
+    
+    const titleMap = {
+        'all': 'All Participants',
+        'notyetverified': 'Not Verified Participants',
+        'cannotbeverified': 'Cannot Be Verified Participants',
+        'verified': 'Verified Participants',
+        'profileNotSubmitted': 'Profile Not Submitted',
+        'consentNotSubmitted': 'Consent Not Submitted',
+        'notSignedIn': 'Not Signed In'
+    };
+
+    const buttonMap = {
+        'all': 'allBtn',
+        'notyetverified': 'notVerifiedBtn',
+        'cannotbeverified': 'cannotVerifiedBtn',
+        'verified': 'verifiedBtn',
+        'profileNotSubmitted': 'profileNotSubmitted',
+        'consentNotSubmitted': 'consentNotSubmitted',
+        'notSignedIn': 'notSignedIn'
+    };
+    
+    removeActiveClass('dropdown-item', 'dd-item-active');
+    removeActiveClass('nav-link', 'active');
+
+    document.getElementById('participants').innerHTML = `<i class="fas fa-users"></i> ${titleMap[type]}`;
+    document.getElementById('participants').classList.add('active');
+    document.getElementById(buttonMap[type])?.classList.add('dd-item-active');
+};
+
+export const updateNavBar = (activeElement) => {
```

Comment:
Good refactor, extracting repeated nav logic from other files into this helper function. Also, I like the above `updateActiveElements` helper function too.